docs/python/python-on-azure.md

```diff
@@ -20,6 +20,6 @@ The following tutorials on the [Python Azure Developer's Center](https://learn.m
 
 Tutorial | Description
 --- | ---
-[Deploy Python web apps to Azure App Service](https://learn.microsoft.com/azure/python/tutorial-deploy-app-service-on-linux-01) | Deploy a web app using the [Azure App Service](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azureappservice) extension.
+[Deploy Python web apps to Azure App Service](https://learn.microsoft.com/en-us/azure/app-service/tutorial-python-postgresql-app?toc=%2Fazure%2Fdeveloper%2Fpython%2Ftoc.json&bc=%2Fazure%2Fdeveloper%2Fbreadcrumb%2Ftoc.json&tabs=flask%2Cwindows%2Cazure-portal%2Cterminal-bash%2Cazure-portal-access%2Cvscode-aztools-deploy%2Cdeploy-instructions-azportal%2Cdeploy-instructions--zip-azcli%2Cdeploy-instructions-curl-bash) | Deploy a web app with PostgreSQL database using the [Azure App Service](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azureappservice) extension.
```

Comment:
That is a scary looking URL. Does it need to have so many query parameters? Maybe they are required for the appropriate tabs to be displayed?
Nit: Remove 'en-us' from URLs so that readers will get the localized version if non-English